README.md

```diff
@@ -53,6 +53,11 @@ mommy knows her little girl can do better~ ❤️
 If you try to make it into an alias, you should prefer pointing it to `cargo-mommy` directly,
 we wouldn't want to break the rustup toolchain picker, now would we?~
 
+If you want to use `cargo-mommy` for not-cargo programs, just set the `CARGO_MOMMY_ACTUAL`
+environment variable to it, for example `CARGO_MOMMY_ACTUAL=cowsay cargo-mommy`~

```

Comment:
Could you flesh this out with an example invocation, I don't follow the full implication